src/material/stepper/_stepper-theme.scss

```diff
@@ -15,6 +16,25 @@
   $warn: map.get($config, warn);
 
   .mat-step-header {
```

Comment:
The HC styles should be in the `_stepper.scss`, not in the theme.

Reply:
I can't find that file

Reply:
Ah sorry, I was referring to this file: https://github.com/angular/components/blob/master/src/material/stepper/stepper.scss